internal/server/server.go

```diff
@@ -15,7 +15,7 @@ import (
 	"time"
 
 	"github.com/gorilla/mux"
-	"github.com/mattermost/mattermost-server/v5/mlog"
+	"github.com/mattermost/mattermost/server/public/shared/mlog"
```

Comment:
@gabrieljackson PTAL on this file as updating the dependency causes syntax error.

Reply:
@stafot I took a look at this and it will probably be a bit of work to fix. Let's merge your other changes and we will do a follow-up to work on the logging.

Reply:
ok I will refactor this PR accordingly and open a ticket to handle the rest.